bindings/python/src/lib.rs

```diff
@@ -974,12 +973,12 @@ fn create_tensor<'a>(
         let dtype: PyObject = get_pydtype(module, dtype, is_numpy)?;
         let count: usize = shape.iter().product();
         let shape = shape.to_vec();
-        let shape: PyObject = shape.into_py(py);
         let tensor = if count == 0 {
             // Torch==1.10 does not allow frombuffer on empty buffers so we create
             // the tensor manually.
             // let zeros = module.getattr(intern!(py, "zeros"))?;
-            let args = (shape.clone(),);
+            let shape: PyObject = shape.clone().into_py(py);
```

Comment:
PyObject doesn'tseem to implement clone anymore therefore we're just cloning the raw Rust object.